nncf/experimental/tensor/functions/numeric.py

```diff
@@ -633,3 +662,86 @@ def diag(a: Tensor, k: int = 0) -> Tensor:
     :return: A tensor with the extracted diagonal.
     """
     return Tensor(diag(a.data, k=k))
+
+
+@functools.singledispatch
+@tensor_guard
+def logical_or(x1: Tensor, x2: Tensor) -> Tensor:
+    """
+    Computes the element-wise logical OR of the given input tensors.
+    Zeros are treated as False and nonzeros are treated as True.
+
+    :param x1: The input tensor.
+    :param x2: The tensor to compute or with.
+    :return: Result of elementwise or operation between input_ and other tensor.
+    """
+    return Tensor(logical_or(x1.data, unwrap_tensor_data(x2)))
+
+
+@functools.singledispatch
+@tensor_guard
+def zero_elements(x: Tensor) -> Tensor:
```

Comment:
Could you share user scenario or links to torch or numpy functions?

Reply:
ok, removed